src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextPaths.cs

```diff
@@ -12,6 +12,8 @@ internal sealed class DependencyContextPaths
         private const string DepsFilesProperty = "APP_CONTEXT_DEPS_FILES";
         private const string FxDepsFileProperty = "FX_DEPS_FILE";
 
+        private static readonly char[] s_semicolon = new[] { ';' };
```

Comment:
Do we even want to cache this? If DependencyContextPaths.Create is only called from GetCurrent, and GetCurrent is only called from that initializer, then that Split call happens only once, and caching this is actually a deoptimization.

Reply:
Yeah, I was just fixing that. 😄. See updated commit.